test/functional/rpc_psbt.py

```diff
@@ -753,16 +752,9 @@ def test_psbt_input_keys(psbt_input, keys):
                 break
         psbt_in = dec["inputs"][input_idx]
         # Calculate the input weight
```

Comment:
nit: not sure if this comment is redundant now since the function name is `calculate_input_weight`